### ISSUE TITLE
Per Client PersistentKeepalive

### ISSUE THREAD

**Issue by J0n4t4n** [vx3r/wg-gen-web#17]
Hey, awesome project, I'm really interested in using it for my Wireguard setup.

Sadly I ran into a little issue that is troublesome for my configuration.

I don't want to use a PersistentKeepalive for most clients, as they are mobile devices that don't require it.
But 2 devices are stationary and host services that I want to have available in the VPN network, thus I need a PersistentKeepalive so that they automatically reconnect after a connection drop.

In short: I would like to have PersistentKeepalive be an option that can be set or overwritten per client instead of just globally.

**Comment by vx3r** [comment]
make sens to me, done

**Comment by vx3r** [comment]
Please reopen if the implementation is not enough
